libs/ledger-live-common/src/transaction/common.ts

```diff
@@ -69,8 +71,11 @@ export const fromTransactionStatusRawCommon = (
 export const toTransactionStatusRawCommon = (
   ts: TransactionStatusCommon
 ): TransactionStatusCommonRaw => ({
-  errors: mapValues(ts.errors, toErrorRaw),
-  warnings: mapValues(ts.warnings, toErrorRaw),
+  errors: mapValues(ts.errors, toErrorRaw) as unknown as Record<string, string>,
```

Comment:
Not perfect but lodash as several signatures for the method mapValues, typescript can't know as of now which one is used
It's not a good practice to say but not "illegal": https://basarat.gitbook.io/typescript/type-system/type-assertion#double-assertion
Any idea how to avoid this ?

Reply:
Why are you adding those types in the first place ? Cause on my VSCode all the types are inferred automatically.

Anyway `mapValues` has multiple typescript signatures using templating, so I guess you should probably take a loot at that:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/29d4592066a1587254230035f5fbb7c5af46ef19/types/lodash/common/object.d.ts#L1518

`mapValues<Record<string, string>, Error>(ts.errors, toErrorRaw)` would be the correct type here I think.